wgsl/index.bs

```diff
@@ -2106,18 +2118,6 @@ equality_expression
         OpINotEqual
         OpFOrdNotEqual
 
-and_expression
-  : equality_expression
-  | and_expression AND equality_expression
-
-exclusive_or_expression
-  : and_expression
-  | exclusive_or_expression XOR and_expression
-
-inclusive_or_expression
-  : exclusive_or_expression
-  | inclusive_or_expression OR exclusive_or_expression
-
 short_circuit_and_expression
   : inclusive_or_expression
```

Comment:
The precedence order is encoded in the order these call each other, so this should also be changed to `equality_expression` if that is the next thing up the precedence list.

Reply:
Done!